test/hotspot/jtreg/compiler/lib/ir_framework/TestFramework.java

```diff
@@ -134,6 +134,8 @@ public class TestFramework {
                     "Xmixed",
                     "server",
                     "Xlog",
+                    "UseAVX",
+                    "UseKNLSetting",
```

Comment:
I don't think we should add these flags to whitelist in these changes - they can affect generated code.
New RFE is filed already to handle such flags: [8289801](https://bugs.openjdk.org/browse/JDK-8289801)

Reply:
Done.